drivers/node/ssh/ssh.go

```diff
@@ -86,6 +87,29 @@ func (s *SSH) IsUsingSSH() bool {
 	return len(os.Getenv("TORPEDO_SSH_KEY")) > 0 || len(os.Getenv("TORPEDO_SSH_PASSWORD")) > 0
 }
 
+// WaitForValidUptime Wait for valid uptime.
+func (s *SSH) WaitForValidUptime(n node.Node) (interface{}, error) {
```

Comment:
Let's not create an extra func for this, we already have this logic on line 92, let's just improve the existing logic to retry and return only when you get the uptime out not empty and no errors

Reply:
Now logic is incorporated in a  single method.